_pages/about.md

```diff
@@ -6,3 +6,24 @@ title: "About"
 The Computational Abilities Knowledge Exchange (CAKE) Network+ is dedicated to providing a sense of community within the Digital Research Infrastructure (DRI) programme and the UK’s wider large-scale computing community. Outward facing and keen to attract new communities, this network is a central hub that brings projects together to enable them to share their successes, achievements and explore how to most effectively work together and leverage each other’s work.
 
 We remove barriers that prevent effective knowledge exchange: taking care of the best practices, tools and resources, so you can focus on connecting, collaborating and driving progress.
+
+## What we provide
```

Comment:
Do we need to mention fellowships, retreats, exchanges...? Maybe add the picture from the presentation?